### DISCUSSION TITLE
Terminal unused container

### DISCUSSION THREAD

**Discussion by mjdNjhNJ** [zed-industries/zed#16019] [Feature Requests]
I have this space between the terminal tabs and terminal output. What is it for and can I get rid of it somehow? I would like my windows to fit as much code as possible and this is unnecessary.

![image](https://github.com/user-attachments/assets/9deb510a-b50b-42a0-a20b-6d6fb6b78c2b)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
